status-handler.js

```diff
@@ -64,8 +64,8 @@ class StatusHandler {
     const ev = this._transform(data, 'event');
     const [, head, current] = await Promise.all([
       StatusEvent.create(ev),
-      StatusHead.findOne(ev),
-      Status.findOne(ev)
+      StatusHead.findOne(ev, { ConsistentRead: true }),
```

Comment:
Should we delay the call to statusHead until we figure out if there is a `current`? So we're not always reading a record we won't need most of the time (unless we're trying to create a new head).

Reply:
@msluther we could but making this function complete faster actually doesnt help our current situation 😭

Reply:
but i guess it could save us money in this new aws world

Reply:
Yeah, I was thinking about saving the extra call / cost / speed. Wouldn't block on it though.